src/pfe/portal/modules/Templates.js

```diff
@@ -109,67 +127,152 @@ module.exports = class Templates {
     return this.projectTemplates;
   }
 
-  async getTemplatesOfStyle(projectStyle) {
-    const allTemplates = await this.getTemplateList();
-    const relevantTemplates = allTemplates.filter(template =>
-      getTemplateStyle(template) === projectStyle
-    );
-    return relevantTemplates;
-  }
-
-  writeRepositoryList() {
-    // Use a callback here so we don't block the response to the request.
-    fs.writeJson(this.repositoryFile, this.repositoryList, { spaces: '  ' }, err => {
-      if (err) {
-        log.error(`Error writing repository list to ${this.repositoryFile}: ${err}`)
-      }
+  // Save the default list to disk so the user can potentially edit it (WHEN CODEWIND IS NOT RUNNING)
+  async writeRepositoryList() {
+    try {
+      await fs.writeJson(this.repositoryFile, this.repositoryList, { spaces: '  ' });
       log.info(`Repository list updated.`);
-    });
+    } catch (err) {
+      log.error(`Error writing repository list to ${this.repositoryFile}: ${err}`)
```

Comment:
i think we should propagate this error to the client because then the repository list won't match what they expected